src/Collection/access.ts

```diff
@@ -25,7 +25,7 @@ export async function hasAccess(
     isCommitteeMember(eth_address),
     isTPCollection(collection)
       ? thirdPartyAPI.isManager(collection.third_party_id, eth_address)
-      : isManager(eth_address, collection),
+      : !!collection.is_published && isManager(eth_address, collection),
```

Comment:
I don't know if I understand this condition:
- Why are we casting a (supposedly) boolean value? might be badly typed
<img width="608" alt="image" src="https://user-images.githubusercontent.com/692077/158732554-7e1e7643-ca43-4440-a037-d76f6c5261db.png">
- For what I can tell, this is so checking for managers makes sense. Meaning that only published collections have managers. But reading the code, shouldn't `isManager()` just return false then?

Reply:
The thing is that isManager should only be called with a merged collection. The code should be something like ```isMergedCollection(collection) && isManager(eth_address, collection)```.
I could add this function elsewhere, but I would love to do it with some type changes to collection first.
See related comment down below.